go-manual/modules/ROOT/pages/query-simple.adoc

```diff
@@ -227,24 +227,37 @@ In other words, there is no guarantee that a write query submitted in read mode
 
 
 [#impersonation]
+[role=label--new-5.14]
 === Run queries as a different user
 
-You can execute a query under the security context of a different user with the callback `neo4j.ExecuteQueryWithImpersonatedUser("<somebodyElse>")`, specifying the name of the user to impersonate.
-For this to work, the user under which the `DriverWithContext` object
-was created needs to have the link:{neo4j-docs-base-uri}/cypher-manual/current/administration/access-control/dbms-administration#access-control-dbms-administration-impersonation[appropriate permissions].
-Impersonating a user is cheaper than creating a new `DriverWithContext` object.
+You can execute a query through a different user with the configuration callback `neo4j.ExecuteQueryWithAuthToken()`.
+Switching user at the query level is cheaper than creating a new `DriverWithContext` object.
+The query is then run within the security context of the given user, and not the one under which the driver was instantiated with (i.e., home database, permissions, etc.). +
```

Comment:
The second half seems a little odd. Is there an extra "under" or "with" or such?